python/tvm/relay/transform/transform.py

```diff
@@ -261,32 +261,39 @@ def LazyGradientInit():
     return _ffi_api.LazyGradientInit()
 
 
-def FoldConstantExpr(expr, mod):
+def FoldConstantExpr(expr, mod, fold_qnn=False):
     """Fold the constant expressions in a Relay program.
     Parameters
     ----------
     expr: Expr
         The expression to fold
     mod: IRModule
         The module the expr lives in (for global calls)
+    fskip: bool
```

Comment:
The argument name here doesn't match with the one in the function. Please review the other ones as well.

Reply:
You are rigth, thank you! I will fix it.